### PR TITLE
Added meeting minutes

### PR DESCRIPTION
**Pull request by xieWright**
Added meeting minutes.

### CODE REVIEW

**Review by criticalglitch — APPROVED** [review]
Looks good

**Review by Lurk390 — APPROVED** [review]
Not much a review needed here, we all helped make this

**Review by bakursait2 — APPROVED** [review]

**Comment by bakursait2** [comment]
@xieWright Can you adjust my name to "Abubakur Sait"

File: Meeting_Minutes.txt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+current: Finish Blocks
+Next: Work on Doxygen
+Abubakur:
+Current: Update small parts in the code.
+Next: Work on Doxygen
+Mohammad:
+Current: work on relation between speed and jump
+Next: Work on Doxygen
+
+20/7/2023 5th standered meeting
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:88]: change date formate mm/dd/yyyy

File: Meeting_Minutes.txt [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+Next: Working on shaders
+Abubakur: Finished the world boundary
+Next: building a tree, textures
+Fix clouds issue
+Rebecca: Finished refactoring the world, player and model classes
+Next: Refactor block class
+Mohammad: organizing comments for speed up and down
+Next: work on jump higher
+
+Today meeting on Tuesday Jul18,2023
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:70]: Do same date formate for all please

File: Meeting_Minutes.txt [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+Next: Work on Doxygen
+Abubakur:
+Current: Update small parts in the code.
+Next: Work on Doxygen
+Mohammad:
+Current: work on relation between speed and jump
+Next: Work on Doxygen
+
+20/7/2023 5th standered meeting
+mass: I finished def higher jump and lower jump and test that, link issue numbers with comments, change def name to be more clear for speed up and speed down.
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:89]: change mass to Mohammad Please

**Review by w1158mxs — CHANGES_REQUESTED** [review]
use same date formate Please MM/DD/YYYY

File: Meeting_Minutes.txt (hunk 1 above)
Comment (xieWright) [reply Meeting_Minutes.txt]: done

File: Meeting_Minutes.txt (hunk 2 above)
Comment (xieWright) [reply Meeting_Minutes.txt]: done

File: Meeting_Minutes.txt [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+Next: Working on shaders
+Abubakur: Finished the world boundary
+Next: building a tree, textures
+Fix clouds issue
+Rebecca: Finished refactoring the world, player and model classes
+Next: Refactor block class
+Mohammad: organizing comments for speed up and down
+Next: work on jump higher
+
+7/18/2023
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:70]: MM/DD/YY format

File: Meeting_Minutes.txt [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+current: Finish Blocks
+Next: Work on Doxygen
+Abubakur:
+Current: Update small parts in the code.
+Next: Work on Doxygen
+Mohammad:
+Current: work on relation between speed and jump
+Next: Work on Doxygen
+
+7/20/2023
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:89]: DD/MM/YY format!

File: Meeting_Minutes.txt [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+Mohammod: I finished def higher jump and lower jump and test that, link issue numbers with comments, change def name to be more clear for speed up and speed down.
+Next: Doxygen
+Mark: finish lighting, make it darker to look like night.
+Next Doxygen
+Abu bakur: finished increase the worlds size, split method generate clouds.
+Waiting Rebeca to merge to add my work , Next Doxygen.
+Mahmoud: I just merged my pull request #83, and next : Doxygen
+Rebecca: going to do more work on pr #85, and going to work on one last code split (isolating the world from the model)
+
+7/22/2023
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:100]: DD/MM/YY format!

File: Meeting_Minutes.txt [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+Mahmoud: Work on Doxygen Editing, Literate programming
+Assigned: Window, Player, Blocks, Utilities 
+Mohammad: Finish up the jump speed issue
+Next: Cleaning up
+Bakursait: Help with Literate programming
+Next: Review pull requests and editing
+Rebecca: Working on splitting on a world file, work on inventory file
+Next: Review pull requests and editing
+
+7/25/2023
```
Comment (w1158mxs) [inline Meeting_Minutes.txt:115]: DD/MM/YY format!

**Review by w1158mxs — APPROVED** [review]
Can you do the other dates and then will be good, but I approve it
use formate DD/MM/YY

**Pull request merged**